recipes/napari-affinities/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "napari-affinities" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-affinities-{{ version }}.tar.gz
+  sha256: 850acfa3d4becb8c701c866c91ddf5922001a78c34c0fc642aff175a33a7e770
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7, <=3.9
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.7, <=3.9
+    - pytorch >=1.3.1
+    - numpy
+    - zarr
+    - magicgui
+    - bioimageio.core
+    - gunpowder
+    - affogato
```

Comment:
This seems problematic since there is a pypi package called `affogato` that is unrelated to the conda forge package `affogato`. I'll leave it as is for now

Reply:
Ahh I understand. Ok then that's fine.